### ISSUE TITLE
Publish chart to public repo

### ISSUE THREAD

**Issue by brandond** [containous/traefik-helm-chart#65]
I'm trying to use this with k3's embedded Tiller and the Helm CRD, but it fails as the chart needs to be hosted in a repo somewhere. Would it be possible to publish this (even in its current status) to a public repository?

**Comment by jimbocoder** [comment]
Big :+1: for this! @brandond in the meantime, I just now forked and repo-ified this project in a ghetto way, but it works.  Here it is as a dependency in `Chart.yaml`:

```
dependencies:
  - name: traefik
    version: "3.3.1"
    repository: https://raw.githubusercontent.com/jimbocoder/traefik-helm-chart/master/tiny-repo
```

Hope it helps you until there's motion on this issue.

**Comment by brandond** [comment]
Thanks! I was just getting started with Helm and didn't really understand the repo format or how to host my own. I ended up educating myself enough to stand up a repo on my GitHub Pages site, which got me what I wanted.

**Comment by alexellis** [comment]
Hi 👋 

I would be willing to help the @containous team publish this helm chart to GitHub Pages or S3, just feel free to reach out directly alex@openfaas.com

Having to clone the repo is a big turn-off and doesn't work well in a helm workflow.

cc @emilevauge 

Alex

**Comment by alexellis** [comment]
If someone in the community or team wants to get to it first, it's relatively simple:

* https://medium.com/@mattiaperi/create-a-public-helm-chart-repository-with-github-pages-49b180dbb417
* https://medium.com/@int128/publish-your-helm-charts-on-github-pages-af439b7ba94f

**Comment by ldez** [comment]
@alexellis Thank you, if you want to contribute on it it will be great :+1: 

We would like to publish the helm chart to GitHub Pages .

Ask me if you need something.

**Comment by ldez** [comment]
I created the `gh-pages` branch.

And you can use the following code for inspiration:

- https://github.com/containous/maesh/blob/2d250776f3edb97da48e7a4b7d2b77cdee886277/Makefile#L104-L108
- https://github.com/containous/maesh/blob/2d250776f3edb97da48e7a4b7d2b77cdee886277/Makefile#L127-L133
- https://github.com/containous/maesh/blob/2d250776f3edb97da48e7a4b7d2b77cdee886277/.travis.yml#L73-L88

**Comment by ldez** [comment]
Closed by #81
